docs/validators/redundancy.md

```diff
@@ -0,0 +1,48 @@
+title: Node redundancy
+---
+
+Each validator node can be backed up by one or several backup nodes in order to 
+provide security of the network and to assure that a node will unlikely get jailed.
+
+The redundancy mechanism relies on the following principle: the owner of the BLS key will start `n` nodes
+with almost the same configuration, and the same `validatorKey.pem` file. The difference between 
+configurations consists on an option inside the `prefs.toml` file.
+That configuration option called `RedundancyLevel` can be used as following:
+
+- a 0 value will represent that the node is the main validator.
+The value 0 will be the default, if the option is missing (as we do not re-write the `prefs.toml` 
+  files during the node's upgrade) will still make that node the main validator: 
+  backwards compatibility holds.
+- a positive value will represent the "order of the back-up node" in the automatic fail-over sequence. 
+  Example: let's suppose we have 3 nodes running with the same key. One with the redundancy level set to 0,
+  one with 1 and one with 3. The node with level 0 will propose and sign upon blocks. The other 2 will 
+  sync (and shuffle out) but will not sign anything. If the main validator fails, the back-up validator 
+  with level 1 will start producing/signing blocks after `level*5` missed rounds. So, after 5 
+  missed rounds by the main validator, the back-up node with level 1 will take the turn. 
+  Now, if the back-up node 1 is down, the second back-up node will start producing/signing blocks 
+  after `3*5 = 15 rounds` after the main validator failed and 10 rounds after the failed back-up node 1 
+  should have been produced a block.
+- a large value for this level option (say 1 million), or a negative value (say -1) will mean that the 
+  backup observers won't get the chance to produce/sign blocks but will sync with the network and 
+  shuffle out just as the main validator will behave.
+
+:::tip
+The backup nodes will advertise on the network a different public key (autogenerated at start-up) and thus, concealing the real public key that will be used when signing the header blocks. 
+:::
+
+:::tip
+If a validator recovers and start signing blocks again, all other validators with higher `RedundancyLevel` will become inactive again.
+:::
+
+:::warning
+Do not use the same redundancy level on more than one node as the nodes with the same `RedundancyLevel` value will start signing blocks in parallel in the same time. The protocol can withstand this kind of things safely but in the near future the BLS key that will perform double signing can get slashed.
+:::
+
+The main reasons for using another BLS key are the following: 
+- the back-up node(s) will not cause BLS signature re-verification when idle.
+- it slightly prevents DDoS attacks as an attacker can not find all IPs behind a targeted BLS public key: 
+  when an attacker takes down the main host, the back-up nodes will advertise the public key when they 
```

Comment:
main host -> Main Validator
back-up nodes -> hot-standby nodes

Reply:
done and done